homeassistant/components/fritz/sensor.py

```diff
@@ -105,12 +113,12 @@ def name(self) -> str:
     @property
     def device_class(self) -> str | None:
         """Return device class."""
-        return str(self._sensor_data[SENSOR_DEVICE_CLASS])
+        return self._sensor_data["device_class"]
 
     @property
     def icon(self) -> str | None:
         """Return icon."""
-        return str(self._sensor_data[SENSOR_ICON])
+        return self._sensor_data.get("icon")
```

Comment:
There's no point in using `get` as `TypedDict` guarantees all keys exist (unless marked with `total=False`).
```suggestion
        return self._sensor_data["icon"]
```